README.md

```diff
@@ -75,7 +75,8 @@ development and community news:
 - Follow us on [Twitter] and [Facebook].
 - Subscribe to the [Mixxx Development Blog][blog].
 - Join the developer [mailing list].
-- Post on the [Mixxx forums][forums].
+- Post on the [Mixxx discord channel][discord].
+- Archive of the [Mixxx forums][forums].
```

Comment:
the old forum was ported to discourse entirely, so we don't need an archive link.

Reply:
for example an old skin thread:
http://www.mixxx.org/forums/viewtopic.php?f=8&t=6052
discourse port:
https://mixxx.discourse.group/t/dark-metal-skin/14096

Reply:
> for example an old skin thread:
> http://www.mixxx.org/forums/viewtopic.php?f=8&t=6052

This shouldn't even work, I guess the redirect is broken again. Ping @rryan

Reply:
FWIW I picked the link from the Dark metal PR

Reply:
Should I remove the link to the archives then?

Reply:
Yes

Reply:
This should be resolved by [da1399f](https://github.com/mixxxdj/mixxx/pull/3116/commits/da1399fa4bf805202b99ab0e8278d13748a63040).